migration/20170224-2-copy-collection-configuration-into-database.py

```diff
@@ -100,12 +100,12 @@ def convert_axis(_db, library):
     collection.external_account_id = library_id
     collection.external_integration.url = url
 
-def convert_one_click(_db, library):
-    config = Configuration.integration('OneClick')
+def convert_rbdigital(_db, library):
+    config = Configuration.integration('RBDigital')
```

Comment:
Leave this bit alone; this is an old migration script that's looking in a JSON configuration document for OneClick credentials. There should not be any sites that need to run this anymore, but if there are that JSON document will still say "OneClick". The rest of the file is  fine though.